README.md

```diff
@@ -1,133 +1,29 @@
-[fouc]: https://en.wikipedia.org/wiki/Flash_of_unstyled_content
-[`next-themes`]: https://github.com/pacocoursey/next-themes
-
-## Font (Optional)
-
-Out of the box, Moai uses [system fonts]. For example, on a modern Mac OS, you
-would see the [San Francisco] typeface. This requires no additional setup
-and is the best solution performance-wise.
-
-However, the for best visual result, we highly recommend the [Inter] typeface.
-To use it, just import the following CSS at your app's root, and Moai will use
-Inter instead:
-
-```tsx
-import "@moai/core/dist/font/remote.css";
-```
-
--   Moai uses the [variable] version of Inter. Check browser compatibility if you
-    need to support old browsers.
--   If your app cannot make network requests, replace "remote" with "local" in the
-    path above to use a bundled version of Inter. This may require a [special
-    setup].
--   It is NOT recommended to use the version on Google Fonts since it lost many
-    [OpenType features].
-
-[system fonts]: https://github.com/moaijs/moai/blob/d3c7c957016b0a8de7838c35fe309384154f9f40/core/src/global.css#L54-L55
-[san francisco]: https://en.wikipedia.org/wiki/San_Francisco_(sans-serif_typeface)
-[inter]: https://rsms.me/inter/
-[variable]: https://en.wikipedia.org/wiki/Variable_font
-[instruction]: https://rsms.me/inter/#usage
-[opentype features]: https://en.wikipedia.org/wiki/List_of_typographic_features
-[special setup]: https://webpack.js.org/guides/asset-management/#loading-fonts
-
-# Use the components
-
-After the setup, just import the components from the package to use. All
-components are exported at the top level:
-
-```tsx
-import { Button } from "@moai/core";
-
-export const Foo = (): JSX.Element => <Button>Hello</Button>;
-```
-
--   Moai is distributed with an [ES6 module] version. If you are using an
-    ES6-aware bundler (such as [Rollup] or [Webpack 2+]), your app won't contain
-    the whole Moai library but only the components that you used, which may
-    significantly reduce your app size.
-
-[es6 module]: https://github.com/rollup/rollup/wiki/pkg.module
-[rollup]: https://rollupjs.org/guide/en/
-[webpack 2+]: https://webpack.js.org/guides/tree-shaking/
-
-# Contributing
-
-(Updating...)
-
-# License
-
-This project is made available under the MIT License
+-   Website: [moaijs.com](https://moaijs.com)
+-   Documentation: [docs.moaijs.com](https://docs.moaijs.com)
+-   NPM: [@moai/core](http://npmjs.com/package/@moai/core)
+-   License: [MIT](./LICENSE)
+
+### Contributing
+
+-   [Contributing Overview](./.github/CONTRIBUTING.md): Report bugs, request features, write code and improve documentation.
+-   [Design Principles](https://docs.moaijs.com/?path=/docs/intro-principles--page): The unique traits that lead Moai's design and development.
+-   [Development](./.github/DEVELOP.md): Codebase overview and local development setup.
+-   [Code of Conduct](./.github/CODE_OF_CONDUCT.md): How we treat each other.
+
```

Comment:
How about putting webuild badge or madeinvietnam badge to be displayed in curated list of webuild? It is a good wait to marketing.

Reply:
Hm interesting, I think I should

Reply:
OK @lqt93 I've just looked around and there are also other badges I'd like to add. Will add them all in another PR